android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -231,6 +259,22 @@ private void createDatePicker() {
         });
     }
 
+    private ArrayList<Item> createItemsFromNotes() throws IllegalArgumentException {
+        String[] lines = notes.getText().toString().trim().split(";");
+        ArrayList<Item> result = new ArrayList<Item>();
+        for (String s : lines) {
+            String[] item = s.split(",");
+            if (2 == item.length) {
+                result.add(new Item(item[0], Float.parseFloat(item[1])));
+            } else if (3 == item.length) {
+                result.add(new Item(item[0], Float.parseFloat(item[1]), Float.parseFloat(item[2])));
+            } else {
+                throw new IllegalArgumentException("Incorrect input format");
+            }
+        }
+        return result;
+    }
+
```

Comment:
I'm sorry, but I think this part is terrible user experience. The user has multiple issues here:
- They cannot see the (quite counter-intuitive) format when starting to type
- The format is not localized. For example, if I enter `Shampoo, 2,00 €, 5` I get a (quite unhelpful) error message. The problem is not only the euro sign, but also the wrong number separator. German users expect to enter amounts in Euros using a dot for number separating.

Let's talk about this in person how to best proceed here :)

Reply:
as a result of our communication, we'll tackle this later on.